mods/sethome/init.lua

```diff
@@ -9,6 +9,14 @@ local S = minetest.get_translator("sethome")
 local homes_file = minetest.get_worldpath() .. "/homes"
 local homepos = {}
 
+local formspec =
+        "size[8,4]" ..
+        "real_coordinates[true]" ..
```

Comment:
I suggest using `formspec_version[2]` `real_coordinates` weren't possible in version < 2 anyway.

Reply:
Done!